### PR TITLE
feat(app): add location select to module calibration wizard

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Introduce a module location select to the module calibration wizard flow that defaults to the slot
that a module is loaded into within a protocol during setup

# Review requests

- launch module calibration from the robot settings or the module card on the robot details page of the desktop app. Should be prompted to select a location for the module, defaulting to the module's default location.
- launch module calibration from the odd protocol setup flow, confirm that you can select deck location, but that the location defaults to the slot that the module was loaded into within the protocol. 

# Risk assessment
low

### CODE REVIEW

**Review by shlokamin — APPROVED** [review]
lgtm! pushed up 2 small changes to get rid of console errors:

1. remove the newline chars in the deck slot location component's path tags
2. add react fragment keys

File: app/src/organisms/ModuleWizardFlows/AttachProbe.tsx [hunk 1]
```diff
@@ -39,6 +37,14 @@ const IN_PROGRESS_STYLE = css`
     margin-top: ${SPACING.spacing4};
   }
 `
+const BODY_STYLE = css`
+  ${TYPOGRAPHY.pRegular};
+
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    font-size: 1.275rem;
+    line-height: 1.75rem;
```
Comment (koji) [inline app/src/organisms/ModuleWizardFlows/AttachProbe.tsx:45]:
```suggestion
    line-height: ${TYPOGRAPHY.lineHeight28};
```

File: components/src/hooks/useSelectDeckLocation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import isEqual from 'lodash/isEqual'
+import { DeckDefinition, getDeckDefFromRobotType } from '@opentrons/shared-data'
+import { RobotCoordinateSpace } from '../../hardware-sim/RobotCoordinateSpace'
+
+import type { ModuleLocation, RobotType } from '@opentrons/shared-data'
```
Comment (koji) [inline components/src/hooks/useSelectDeckLocation/index.tsx:6]: can we move this line after DeckSlotLocation import?

**Review by koji — APPROVED** [review]
The wizard worked as expected

File: app/src/organisms/ModuleWizardFlows/AttachProbe.tsx (hunk 1 above)
Comment (b-cooper) [reply app/src/organisms/ModuleWizardFlows/AttachProbe.tsx:45]: will be addressed by @shlokamin 's following PR

File: components/src/hooks/useSelectDeckLocation/index.tsx (hunk 2 above)
Comment (b-cooper) [reply components/src/hooks/useSelectDeckLocation/index.tsx:6]: will be addressed in @shlokamin 's following PR

**Pull request merged**